ide/editor.settings/src/org/netbeans/api/editor/settings/SimpleValueNames.java

```diff
@@ -295,7 +295,7 @@ public final class SimpleValueNames {
     public static final String COMPLETION_AUTO_POPUP = "completion-auto-popup"; // NOI18N
 
     /**
-     * Whether the code completion query search will be case  sensitive
+     * Whether the code completion query search will be case-sensitive
      * Values: java.lang.Boolean
      */
     public static final String COMPLETION_CASE_SENSITIVE = "completion-case-sensitive"; // NOI18N
```

Comment:
It might make sense to put the option name to the API, as is done for other options.

Reply:
I didn't quite understand this suggestion. I refrained from adding to `SimpleValueNames` because the comment in `EditorPreferenceKeys` mentioned that the latter is meant for *private* keys, while the former for *public*, *devel* or *friend* keys.
I can move the key here if I misunderstood. Thanks.

Reply:
I have removed the unrelated grammatical change in this file. Thanks.